<!--
  The turtlebot people (or whatever) follower nodelet.
  This is a test version that starts LIDAR independently.
 -->
<launch>
  <param name="robot/name" value="SarahBella" type="str"/>
  <param name="robot/type" value="turtlebot3" type="str"/>
  <param name="robot/application" value="follow" type="str"/>
  <param name="robot/platform" value="RaspberryPi" type="str"/>
  <param name="robot/behavior" value="follow" type="str"/>
  <!-- These metrics influence the follower algorithm -->
  <param name="/follow/stop_distance" value="0.3" />
  <param name="/follow/follow_distance" value="0.5" />
  <param name="/follow/max_speed" value="0.5" />
  <param name="/follow/min_speed" value="0.01" />

  <include file="$(find turtlebot3_bringup)/launch/turtlebot3_robot.launch"/>
  <node pkg="follower" type="follow_object.py" name="sb_follow" output="log">
  </node>

</launch>
